src/SourceBuild/content/build.proj

```diff
@@ -25,6 +27,17 @@
              StopOnFirstFailure="true" />
   </Target>
 
+  <Target Name="AfterBuild" AfterTargets="Build">
+    <PropertyGroup>
+      <AssetManifestsLocalStorageDir>$(ArtifactsObjDir)</AssetManifestsLocalStorageDir>
```

Comment:
IMO a manifests dir would be nice here.

Reply:
I put the `AssetManifestsIntermediateDir` which was already defined